src/components/layouts/navigation.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import BottomNavigation from '@material-ui/core/BottomNavigation';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+import RestoreIcon from '@material-ui/icons/Restore';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+
+const useStyles = makeStyles({
+  // change colors: https://material-ui.com/customization/components/#overriding-with-classes , changing colors doesn't work...
+  root: {
```

Comment:
I know for a fact that changing colours works. I've done so myself, please check this out:

```JS
  const theme = createMuiTheme({
        overrides: {
            MuiBottomNavigationAction: {
                root: {
                    fontSize: '20px',
                    "&$selected": { 
                        paddingTop: 10,
                        color: "#ffffff",
                        backgroundColor: "#c51162"
                    }
                }
            }
        }
    });
``` 
Check the documentation for more information on how you can override the standard colourscheme:
https://material-ui.com/api/bottom-navigation/

Reply:
You can then wrap your `BottomNavigation` in a `ThemeProvider` component:
```JS
<ThemeProvider theme={theme}>
            <BottomNavigation value={value} onChange={handleChange} className={classes.root}>
                <BottomNavigationAction label="Home" value={0}
                    icon={
                        <Badge anchorOrigin={{
                            vertical: 'top',
                            horizontal: 'left',
                        }}
                            color="primary" overlap="circle" badgeContent={2}>
                            <FontAwesomeIcon icon={faHome} />
                        </Badge>
                    } />
                <BottomNavigationAction label="Projects" value={1} icon={<FontAwesomeIcon icon={faProjectDiagram} />} />
                <BottomNavigationAction label="Settings" value={2} icon={<FontAwesomeIcon icon={faCog} />} />
            </BottomNavigation>
        </ThemeProvider>
```